### ISSUE TITLE
Hi ! About KITTI val performance

### ISSUE THREAD

**Issue by ljwwwiop** [chaytonmin/Voxel-MAE#31]
<img width="435" alt="image" src="https://github.com/chaytonmin/Occupancy-MAE/assets/45536607/cbe04db7-a900-4644-8918-5ab212616a32">
<img width="1089" alt="image" src="https://github.com/chaytonmin/Occupancy-MAE/assets/45536607/6342d121-e949-4e42-af4c-fb7ea9762edc">

Thanks for the ideas on pre-training methods.
However, my experimental results on the KITTI validation set differ from the results in Table V in the paper. I used the results from the 30th epoch of the MAE, and I wonder if this is the reason for the discrepancy?
